### ISSUE TITLE
localhost

### ISSUE THREAD

**Issue by daniel030794** [cami-la/sacola-api_IFOOD_DEV_WEEK#7]
Boa tarde cami, quando vou abrir o meu localhost a parte da jdbc url não abre com a mesma que colocamos na ide, e ai da erro quando tento fazer o test conection e nao conecta também, poderia me ajuda com isto?

**Comment by cami-la** [comment]
![2022-09-29_18-56](https://user-images.githubusercontent.com/64323124/193149456-7dd1ec7b-cef2-4ce3-8348-d33f6d4cf62a.png)

Estão essas mesmas configurações? Confere por favor.

**Comment by daniel030794** [comment]
sim Cami as mesmas porem mesmo eu trocando a url manualmente nao vai.

**Comment by cami-la** [comment]
Tira um print, por favor. 
Aí consigo entender melhor! 🫶

**Comment by cami-la** [comment]
Oi, @daniel030794. Tudo bem?
Então, as anotações estão erradas. 🥹

@Data
@Entity
@Builder

Mas de qualquer forma, vai tentando que você consegue.

**Comment by daniel030794** [comment]
Oi cami, mas o que nao vi na aula foi essas anotações como chegar nela , meu codigo nao rodava foi quando vim no seu github e percebi as anotações que elas consequentemente adiciona no codigo os imports ne

**Comment by cami-la** [comment]
Isso, essas anotações servem para você dizer ao seu programa o que você quer.
Através dessas anotações é que você consegue importar as classes. Note que as importações do **Lombok**, só funcionam porque nós adicionamos a dependência do **Lombok** no arquivo **build.grade**.

**Comment by daniel030794** [comment]
Sim entendi Cami mas porque essas notações não entram automático no meu código, igual comentei eu só consegui rodar elas após vim verificar o seu código e ver essas notações. Até então não tinha noção de onde e como fazer Las

**Comment by cami-la** [comment]
Certo, aí o que você faz. Todas as vezes que você colocar uma anotação, você deixa o cursor do editor em cima da anotação e pressiona Ctrl + Alt aí vai aparecer uma caixinha perguntando se você quer fazer o import.

**Comment by daniel030794** [comment]
Entendi.. mas digo assim, na aula não havia essas notações, como eu teria que chegar nela no caso sem conhece Las .. porque como te disse eu só soube que necessitavam delas quando vim tirar dúvidas no seu código. Até então e na aula não mostrava elas .

**Comment by cami-la** [comment]
Todas as anotações eu coloquei durante a aula....

**Comment by georgiasilvabs** [comment]
Bom dia, estou com o mesmo problema quando rodo o código vem no JDBC URL: jdbc:h2:~/test. Mudei manualmente para jdbc:h2:mem:testdb, mas aparece o seguinte erro "[Database "mem:testdb" not found, either pre-create it or allow remote database creation (not recommended in secure environments) [90149-214]](http://localhost:8080/h2-console/test.do?jsessionid=36a668d3378ef1e67b30fdf7d83b70e3#) 90149/90149 [(Help)](https://h2database.com/javadoc/org/h2/api/ErrorCode.html#c90149)
![h2](https://user-images.githubusercontent.com/105814627/193952092-e739bdc0-803b-4665-b86c-94ef51e433ff.png)
![h1](https://user-images.githubusercontent.com/105814627/193952105-b388c598-5ddb-452a-87f8-46e780ce1891.png)

**Comment by cami-la** [comment]
Oi, @georgiasilvabs. Tudo bem? (:
Então, veja que a identação do arquivo application.yml está diferente do meu arquivo. 

Copia o meu arquivo data.sql e cola no seu projeto que vai dar certo. Mas antes de copiar e colar, dá uma olhada na identação: https://github.com/cami-la/sacola-api_IFOOD_DEV_WEEK/blob/master/src/main/resources/application.yml

![2022-10-09_21-08](https://user-images.githubusercontent.com/64323124/194785722-720fd5f2-50ee-4294-8d9e-8f8787a77810.png)

**Comment by daniel030794** [comment]
Cami eu consegui rodar porém após ter inserido a linha pathmatch: no vídeo
aula não tinha mas copiei do seu código só github eai conseguir entrar no
banco de dados.

Em qui, 29 de set de 2022 23:25, Camila Cavalcante ***@***.***>
escreveu:

**Comment by cami-la** [comment]
Oi, @daniel030794. Tudo bem?
Então, essa configuração que você citou não tem relação com o banco de dados não, tá?
Tem relação com a documentação da API com o Swagger que nós vamos utilizar no último dia.
Logo, deveria funcionar sem ela.

**Comment by Jossondy** [comment]
Boa tarde Camila. Tudo bem com vc?
Meu API eu conseguiu ver ele no postman mas não conseguiu abrir ainda no localhost.
Favor, me ajuda🙏🏽!

Jossondy Princetile

https://github.com/Jossondy


Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by cami-la** [comment]
Detalha mais um pouco o erro, @Jossondy. Manda um print do seu projeto no IntelliJ.

**Comment by Jossondy** [comment]
________________________________
De : Camila Cavalcante ***@***.***>
Envoyé : mardi 11 octobre 2022 13:29
À : cami-la/sacola-api_IFOOD_DEV_WEEK ***@***.***>
Cc : Jossondy Princetile ***@***.***>; Mention ***@***.***>
Objet : Re: [cami-la/sacola-api_IFOOD_DEV_WEEK] localhost (Issue #7)


Detalha mais um pouco o erro, @Jossondy<https://github.com/Jossondy>. Manda um print do seu projeto no IntelliJ.

**Comment by Jossondy** [comment]
@cami-la

**Comment by Jossondy** [comment]
>

![Localhost](https://user-images.githubusercontent.com/114502801/195164950-24fecd99-c5e1-4a66-8507-5b049eb30ab2.PNG)
![Capturar1](https://user-images.githubusercontent.com/114502801/195164952-7d0dc0fd-2767-49f1-b35e-5088ffc2b616.PNG)

**Comment by Jossondy** [comment]
> Detalha mais um pouco o erro, @Jossondy. Manda um print do seu projeto no IntelliJ.
@cami-la 
![Localhost](https://user-images.githubusercontent.com/114502801/195165074-cc8204d0-6fe2-42d4-9ea4-8ea1317f15f7.PNG)
![Capturar1](https://user-images.githubusercontent.com/114502801/195165075-a3bf1a6f-8fb5-412e-a57c-f0430787117f.PNG)

**Comment by cami-la** [comment]
Certo.
Agora confere por favor, se arquivo **application.yml** está escrito da mesma forma com a mesma identação que esse aqui: https://github.com/cami-la/sacola-api_IFOOD_DEV_WEEK/blob/master/src/main/resources/application.yml

**Comment by Jossondy** [comment]
@cami-la it's the same, eu fiz o copy/paste do seu arquivo mesmo.

**Comment by Jossondy** [comment]
> Certo. Agora confere por favor, se arquivo **application.yml** está escrito da mesma forma com a mesma identação que esse aqui: https://github.com/cami-la/sacola-api_IFOOD_DEV_WEEK/blob/master/src/main/resources/application.yml
![App yml](https://user-images.githubusercontent.com/114502801/195175347-b9308827-fa4c-4eef-b968-6782bcb0adb7.PNG)
Confere!

**Comment by cami-la** [comment]
Ah, já sei. 
A url está: **localhost:8081/h2.console** e a URL correta é:  l**ocalhost:8081/h2-console**

**Comment by Jossondy** [comment]
@cami-la ptchiu 😁😁😁😁😁😁 deu certo. Muito obrigado
